packages/hooks/src/validateNamespace.js

```diff
@@ -13,13 +13,8 @@ function validateNamespace( namespace ) {
 		return false;
 	}
 
-	if ( ! /^[a-zA-Z][a-zA-Z0-9_.\-/]*$/.test( namespace ) ) {
```

Comment:
The documentation for this function still reflects the `vendor/plugin/function` format. It's also a bit of a misnomer to call this a namespace validation, since at this point it's really just checking that the function name takes a particular form (there is no namespace).